docs/blog/design-philosophy/index.qmd

```diff
@@ -0,0 +1,245 @@
+
+![](./composition_of_a_table_in_GT.png)
+
+<p style="font-size: 12px;">A schematic with the complete set of table components that can be utilized in **Great Tables**.</p>
+
+If you followed our advice to look at the *Census Manual* (or just glanced at our excerpts-based graphic), you might have noticed that the term **Stub** is also in our table schematic. We thought it was too good a name to *not* borrow. You won't find too many mentions of that term elsewhere; others in the table business will use it sparingly, if at all. But that bit of terminology is perfect for the function that component has: to store information that identifies and explains the rows (as well as labels that deal with groupings of rows). The **Stub Head**, while small, is an important addendum. It functions to either label the elements in the stub (below), the elements in the column labels (across), or both. It can even not be present at all.
+
+The table **Heading** is all important to us, and far too many software applications that allow for summary table creation make it difficult or impossible to include a heading. The **Footer** is equally important but is unfortunately often neglected. It provides a place for table notes and will only be present in **Great Tables** if table notes do exist.
+
+The **Spanner Labels** are pieces of text that span across **Column Labels**, allowing for some grouping of columns where it makes sense. This is important for the readability of tables, especially wider tables having many columns.
+
+Aside from a table's stuctural considerations, we also believe that the data values contained within the cells often require formatting before presentation, since the representation for computing isn't always optimal for readers of a table. To that end, we provide a large collection of formatting methods that are purpose-built for each type of formatting operation. While focused on simplicity they are also powerful, giving you the ability to customize values to your liking. We want users all over the world to be able to use **Great Tables** successfully so we built in locale-based formatting options, based on the latest guidance from the [Unicode CLDR](https://cldr.unicode.org).
+
```

Comment:
Another "i suspect you don't want to add this" but in gt docs,
but I like this thin image at about here. 

https://gt.rstudio.com/reference/figures/gt_workflow_diagram.svg

You'd need to modify a few things, but for visual learners, it's sumarizes nicely what you said above. Take your table, and great_tables helps you convert that into a great_table object with package modifiers, and gets it ready for export.